apis/Google.Devtools.Aspnet/Google.Devtools.Aspnet/project.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": "1.0.0-alpha01-*",
```

Comment:
If you make this 1.0.0-alpha00-* that will work better for when we make the first release 1.0.0-alpha01, in terms of CI builds. (We up the version number just before releasing.) Or you can leave it, and the first published release will just be 1.0.0-alpha02, of course.